lesson15/index.js

```diff
@@ -1,4 +1,4 @@
-const main = document.getElementsByTagName('main')[0];
+const main = document.getElementById('js-main');
 const ul = document.createElement('ul');
 const url = 'https://jsondata.okiba.me/v1/json/xL3LX210624191849';
 const fetchBtn = document.getElementById("js-btn-fetch");
```

Comment:
nits:
`fetchBtn` is not used anywhere.
Is this definition necessary?

Reply:
Thanks for the comment!
I have removed unnecessary constant definitions.

1a17737